package.json

```diff
@@ -93,7 +93,11 @@
     "timezones.json": "1.6.1",
     "tinycolor2": "1.4.2",
     "typescript": "4.3.4",
-    "zen-observable": "0.8.15"
+    "zen-observable": "0.8.15",
+    "@vvo/tzdb": "6.48.0",
+    "d3": "7.4.4",
+    "geojson": "0.5.0",
+    "topojson-client": "3.1.0"
```

Comment:
What are all these needed for? This seems like a lot of new dependencies to add for just this feature

Reply:
Looks like there are all being used for the map stuff